apps/pie-storybook/stories/pie-toast.stories.ts

```diff
@@ -2,7 +2,7 @@ import { html } from 'lit';
 import { ToastProps, defaultProps, variants } from '@justeattakeaway/pie-toast';
 import { action } from '@storybook/addon-actions';
 import { type StoryMeta } from '../types';
-import { createStory } from '../utilities';
+import { createStory, TemplateFunction } from '../utilities';
```

Comment:
```suggestion
import { createStory, type TemplateFunction } from '../utilities';
```
And also for `ToastProps` above, please